### PR TITLE
register and login - validation and other changes

### PR DESCRIPTION
**Pull request by miriyak**

### CODE REVIEW

File: controllers/auth.controller.js [hunk 1]
```diff
@@ -34,14 +51,31 @@ router.post('/login', function (req, res) {
...
+            res.json({ isLogged: false, message: "הכניסה נכשלה. נתונים שגויים", user: user });
         }
     });
 
     
 });
+
+
+router.post('/logout', function (req, res) {
+
```
Comment (navacohen90) [inline controllers/auth.controller.js:67]: is it necessary?

File: controllers/auth.controller.js [hunk 2]
```diff
@@ -34,14 +51,31 @@ router.post('/login', function (req, res) {
...
+    //req.session_state.reset();
+    //res.redirect('/');
+
+    
+});
+router.post('/getuser', function (req, res) {
+
```
Comment (navacohen90) [inline controllers/auth.controller.js:74]: is it necessary?

File: controllers/auth.controller.js [hunk 3]
```diff
@@ -10,21 +11,37 @@ var userSchema = require('../models/user');
...
+                    res.json({ isAdded: false, message: "כתובת המייל קיימת במערכת למשתמש אחר", user: user });
+                    flagAdded = false;
+                }
+                else {
+                    user.save();
+                    res.json({ isAdded: true, message: "משתמש נוסף בהצלחה", user: user });
+
+                }
+        
+            });
```
Comment (navacohen90) [inline controllers/auth.controller.js:42]: :+1:

File: controllers/auth.controller.js (hunk 2 above)
Comment (miriyak) [inline controllers/auth.controller.js:74]: Yes, for the next stage - save the user in session, and enable logout

**Pull request merged**